open-metadata-resources/open-metadata-deployment/docker/configure/Dockerfile

```diff
@@ -15,6 +15,12 @@ LABEL org.label-schema.docker.params = "SERVICE=the name of the k8s service to c
 
 # Install utilities we will use to configure containers
 RUN apk --no-cache add bash curl git jq openldap-clients postgresql-client mariadb-client && apk --no-cache update && apk --no-cache upgrade
-COPY dist/check-availability.sh /check-availability.sh
+
+RUN groupadd -r config -g 1000 && useradd --no-log-init -r -g config -u 1000 -d /opt/config config
```

Comment:
groupadd not present, resulting in a build failure
```
[INFO] Step 12/16 : RUN groupadd -r config -g 1000 && useradd --no-log-init -r -g config -u 1000 -d /opt/config config
[INFO]
[INFO]  ---> Running in 7f69ba971f00
[INFO] /bin/sh: groupadd: not found
[INFO]
[ERROR] The command '/bin/sh -c groupadd -r config -g 1000 && useradd --no-log-init -r -g config -u 1000 -d /opt/config config' returned a non-zero code: 127
[WARNING] An attempt failed, will retry 1 more times
org.apache.maven.plugin.MojoExecutionException: Could not build image
```

Reply:
The fix here would be to add the shadow package (ie apk add shadow -- or more likely add to prior add.... I notice the atlas image already ends up including groupadd). Or could use addgroup. I'd go with shadow.

Reply:
will apply that change locally & do a full rebuild to test before merge (assuming above fix)